copier.yaml

```diff
@@ -11,7 +11,7 @@ _envops:
   variable_start_string: "{{"
 
 _tasks:
-  - '(cd apps/{{api}}; npx npm-add-script -f -k bootstrap -v "talos put --api \${{api|upper}}_BASE --endpoint \$SPARQL_ENDPOINT --user \$SPARQL_USER --password \$SPARQL_PASSWORD")'
+  - '(cd apps && cd {{api}} && npx npm-add-script -f -k bootstrap -v "talos put --api ${{api|upper}}_BASE --endpoint $SPARQL_ENDPOINT --user $SPARQL_USER --password $SPARQL_PASSWORD")'
```

Comment:
Removing slashes causes variable expansion. Try inverting the quotation and how it works on windows

```suggestion
  - "(cd apps && cd {{api}} && npx npm-add-script -f -k bootstrap -v 'talos put --api ${{api|upper}}_BASE --endpoint $SPARQL_ENDPOINT --user $SPARQL_USER --password $SPARQL_PASSWORD')"
```

Reply:
![Obraz bez nazwy](https://user-images.githubusercontent.com/23079007/161436787-e3b6a95c-fb34-409f-92da-e383b6bd89bb.png)
Sadly that, in turn, makes it wrong on the Windows platform .. 😢 
I got stuck on this one tbh as it's either not working on one system or the other ... I came up with a bit of a hacky way like so: https://github.com/hypermedia-app/creta-template-api/pull/11/commits/9aee66817afe5af63bda8f60af454b815ff6dc08 (and then improved after checking on unix too: https://github.com/hypermedia-app/creta-template-api/pull/11/commits/f224d232b954454b1047d541a02f5444603a42a8) @tpluscode could you have a check please if this works as expected on macOS ? 👍